### DISCUSSION TITLE
Does anyone else ever see a red ghost icon in their window page title?

### DISCUSSION THREAD

**Discussion by elizdlt202** [laravel/nova-issues#5834] [Q&A]
This red ghost icon keeps appearing for me and I have yet to find any discussion on it anywhere...
Can someone help me confirm its' cause?

Thanks

![Screen Shot 2023-08-23 at 2 20 04 PM](https://github.com/laravel/nova-issues/assets/49735067/e4c169b5-03d3-4af6-b331-fa8a053f3d53)

**Comment by ziming** [comment]
I think I saw it before too but not able to reproduce it.

**Reply by elizdlt202** [reply]
thanks for commenting!

**Comment by HenkPoley** [comment]
I'm seeing it on just a local Laravel website in Microsoft Edge on my development laptop.

**Comment by Denobraz** [comment]
This is icon of bug page. It just cached in your browser.
